deploy_config_generator/output/kube_common.py

```diff
@@ -0,0 +1,303 @@
+import copy
+
+from deploy_config_generator.utils import underscore_to_camelcase
+from deploy_config_generator.output import OutputPluginBase
+
+METADATA_FIELD_SPEC = dict(
+    name=dict(
+        type='str',
+        required=True,
+    ),
+    labels=dict(
+        type='dict',
+    ),
```

Comment:
```suggestion
    ),
    annotations=dict(
        type='dict',
    ),
    namespace=dict(
        type='dict'
    ),
```

Reply:
Ah yes, I do need to finish fleshing out the fields for the metadata.

Reply:
Added